src/stylesheets/core/_grid.scss

```diff
@@ -2,109 +2,110 @@
 .usa-grid-full {
   @include outer-container();
   max-width: $site-max-width;
+}
 
-  @include media($medium) {
-    .usa-width-one-whole {
-      @include span-columns(6);
-    }
-    .usa-width-one-half {
-      @include span-columns(3);
-    }
-    .usa-width-one-third {
-      @include span-columns(2);
-    }
-    .usa-width-two-thirds {
-      @include span-columns(4);
-    }
-    .usa-width-one-fourth {
-      @include span-columns(3);
+@include media($medium) {
+  .usa-width-one-whole {
+    @include span-columns(6);
+  }
 
-      &:nth-child(2n) {
-        margin-right: 0;
-      }
-    }
-    .usa-width-three-fourths {
-      @include span-columns(6);
-    }
-    .usa-width-one-sixth {
-      @include span-columns(2);
+  .usa-width-one-half {
+    @include span-columns(3);
+  }
+  .usa-width-one-third {
+    @include span-columns(2);
+  }
+  .usa-width-two-thirds {
+    @include span-columns(4);
+  }
+  .usa-width-one-fourth {
+    @include span-columns(3);
 
-      &:nth-child(3n) {
-        margin-right: 0;
-      }
+    &:nth-child(2n) {
+      margin-right: 0;
     }
-    .usa-width-five-sixths {
-      @include span-columns(5);
-    }
-    .usa-width-one-twelfth {
-      @include span-columns(2);
+  }
+  .usa-width-three-fourths {
+    @include span-columns(6);
+  }
+  .usa-width-one-sixth {
+    @include span-columns(2);
 
-      &:nth-child(3n) {
-        margin-right: 0;
-      }
+    &:nth-child(3n) {
+      margin-right: 0;
     }
   }
+  .usa-width-five-sixths {
+    @include span-columns(5);
```

Comment:
I think this should be `@include span-columns(4);`  see recent change here: https://github.com/18F/web-design-standards/blob/18f-pages-staging/src/stylesheets/core/_grid.scss#L36